terraform/init.tf

```diff
@@ -0,0 +1,43 @@
+terraform {
+  required_version = ">= 1.2.2"
+  backend "azurerm" {}
+  required_providers {
+    azurerm = "~> 3.11.0"
+    sendgrid2 = {
+      source  = "anna-money/sendgrid"
+      version = "1.0.4"
+    }
+    sendgrid = {
+      source  = "Trois-Six/sendgrid"
+      version = "0.1.6"
+    }
```

Comment:
could you explains why there's two terraform providers?

Reply:
so - if we do not keep the original provider, terraform throws error (see below link), I think we will able to remove TROIS-SIX provider after we apply changes.
https://dev.azure.com/hmcts/PlatformOperations/_build/results?buildId=339921&view=logs&j=32bb70dc-ef50-5fbc-2009-ecb74cbf2521&t=5a64c744-dee6-575c-add7-b88ac4edd549